sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 ## Release History
 
-## 4.9.0-beta.1 (Unreleased)
+### 4.9.0 (2020-12-11)
+#### New Features
+* Added Beta API for Bulk Operations.
+* Added `getRegionsContacted` API in `CosmosDiagnostics`.
```

Comment:
`Set<URI> CosmosDiagnostics.getRegionsContacted()`
I might have missed this earlier in the PR.

two comments:
1. For `CosmosClientBuilder.endpoint(String)` we pass the endpoint as String not URI, but in the above newly added API we return the endpoint as URI. seems inconsistent to me.
2. For other apis when we return a collection we return Iterable not Set.

Can we tag this new API with Beta annotation for now. So if we can have the option of changing it later?

Reply:
Makes sense, will add the beta annotation to it.

Reply:
Done.